analyzer/server.go

```diff
@@ -47,9 +51,15 @@ type Server struct {
 	GraphServer         *graph.Server
 	FlowMappingPipeline *mappings.FlowMappingPipeline
 	Storage             storage.Storage
+	FlowTable           *flow.FlowTable
+}
+
+func (s *Server) flowExpire(f *flow.Flow) {
+	s.FlowTable.Remove(f)
```

Comment:
Yes this line should not be here, see my comment : https://github.com/redhat-cip/skydive/pull/31/files#r51262801